survey/default.nix

```diff
@@ -1709,11 +1709,15 @@ let
                  ])
                ]);
         in
-          final.lib.mapAttrs
+          (final.lib.mapAttrs
             (name: value:
               if (isProperHaskellPackage value && isExecutable value) then statify value else value
             )
-            super
+            super) // {
```

Comment:
I was hoping I could get rid of this map, as the `callPackage` override would take over its functionality. That doesn't appear to happen, though, and leaving this out will cause any regular executables to be dynamically linked.